src/pages/workspaces/workspace/Data.js

```diff
@@ -345,6 +345,24 @@ class EntitiesContent extends Component {
     ])
   }
 
+  renderOpenInDataExplorerButton() {
+    const { workspace: { workspace: { workspaceId } } } = this.props
+    const { selectedEntities } = this.state
+
+    return h(Fragment, [
+      buttonPrimary({
+        disabled: _.size(selectedEntities) !== 1,
+        tooltip: _.size(selectedEntities) <= 1 ?
+          'Select a cohort to open in Data Explorer' :
+          'Select exactly one cohort to open in Data Explorer',
```

Comment:
nit: Do we need two slightly different messages, or could they be collapsed into one? Also, I don't think the tooltip is needed when the button is enabled.

Reply:
Removed tooltip when button is enabled.

I think the two messages are useful. If more than one cohort is selected, I don't think former is sufficient to explain why button is disabled. But former is more natural -- if no cohort is selected, I like former better than latter.